src/openforms/js/components/admin/form_design/MissingTranslationsWarning.js

```diff
@@ -0,0 +1,192 @@
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import MessageList from 'components/admin/MessageList';
+import Modal from 'components/admin/Modal';
+import {ChangelistColumn, ChangelistTable} from 'components/admin/tables';
+import jsonScriptToVar from 'utils/json-script';
+
+import {extractMissingComponentTranslations} from './MissingComponentTranslationsWarning';
+
+const extractMissingTranslations = (translations, tabName, fieldNames, fallbackFields) => {
+  const labelMapping = jsonScriptToVar('label-mapping');
+  const languages = jsonScriptToVar('languages');
```

Comment:
This is not something you should call during component renders because they perform DOM manipulation.